org.OpenT2T.Sample.SuperPopular.Thermostat/Nest Thermostat/js/README.md

```diff
@@ -0,0 +1,92 @@
+access_token : 
```

Comment:
Oops.. this must be removed.. I had saved this for testing purposes.

---
In reply to: [68174201](https://github.com/openT2T/translators/pull/22#discussion_r68174201) [](ancestors = 68174201)